docs/source/ops.rst

```diff
@@ -6,7 +6,7 @@ torchvision.ops
 :mod:`torchvision.ops` implements operators that are specific for Computer Vision.
 
 .. note::
-  Those operators currently do not support TorchScript.
+  As now all operators support TorchScript.
```

Comment:
Maybe I would phrase this as something like
```suggestion
  All operators have native support for TorchScript.
```

Reply:
Thx, agree with your suggestion